.github/workflows/upload-delegates.yml

```diff
@@ -2,18 +2,20 @@ on:
   push:
     branches:
       - master
-
+    paths:
+      - governance/delegates/**/*
+      - '!governance/delegates/meta/delegates.json'
```

Comment:
Does this mean to ignore changes to this file?

Reply:
yeah, it won't trigger the workflow after that file is updated (because itself updates that file, and otherwise it enters a loop of PRs)